.changeset/nice-ligers-type.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixes `at-rule-no-unknown` false positives for `@scroll-timeline`
```

Comment:
[suggest] Please update the prefix:

```suggestion
Fixed: `at-rule-no-unknown` false positives for `@scroll-timeline`
```

See also:

https://github.com/stylelint/stylelint/blob/797381ced3190aa0c78bff8fb892572caadf4305/docs/maintainer-guide/pull-requests.md?plain=1#L22-L24

Reply:
Ah - my mistake! I'll also adjust my other PR with this. Still new to the changeset tool!